lib/parse/base.js

```diff
@@ -159,8 +159,6 @@ Base.prototype.getFileSource = function (filename) {
 			header: false,
 			bare: true
 		});
-	} else if (filename.match(this.jsxExtRegEx)) {
```

Comment:
Yes, should probably be removed. And `react-tools` is deprecated per: https://facebook.github.io/react/blog/2015/06/12/deprecating-jstransform-and-react-tools.html.

Incidentally, are you a maintainer on `madge`, or - what's your interest in it?